src/PublicApiGenerator/ApiGenerator.cs

```diff
@@ -1,3 +1,10 @@
+// In case this file is included in a project with the <Nullable>enable</Nullable> csproj property (which sets the
+// /nullable:enable compiler option):
```

Comment:
This comment should have included a `<auto-generated/>` line if the file is meant to be imported effectively as generated code (outside the normal developer workflow). Generated files _default_ to `#nullable disable` even in projects with nullable reference types enabled, so this header would have corrected the warnings you saw.

Reply:
Excellent, thank you!